libs/payments/ui/src/lib/client/components/PurchaseDetails/en.ftl

```diff
@@ -2,6 +2,9 @@
 
 next-plan-details-header = Product details
 next-plan-details-list-price = List Price
+# $productName (String) - The name of the product, e.g. Mozilla VPN
+# $interval (String) - The billing cycle interval
+plan-details-product-prorated-price = { $productName } { $interval } (Prorated)
```

Comment:
@mozilla/fxa-l10n, we will phrase this how it is phrased on the Upgrade page unless advised otherwise.

For reference, on the confirmation page, if there is no prorated adjustment to the price, the price is listed as "List Price" see screenshot [6]. When there is a prorated adjustment to the price (see screenshot [5].